TransformationSystem/Client/TransformationClient.py

```diff
@@ -3,13 +3,14 @@
 __RCSID__ = "$Id$"
 
 from DIRAC                                                         import S_OK, gLogger
-from DIRAC.Core.Base.Client                                        import Client
+from DIRAC.Core.Base.Client import Client, createClient
 from DIRAC.Core.Utilities.List                                     import breakListIntoChunks
 from DIRAC.ConfigurationSystem.Client.Helpers.Operations           import Operations
 
-class TransformationClient( Client ):
 
-  """ Exposes the functionality available in the DIRAC/TransformationHandler
+@createClient('TransformationSystem/TransformationManager')
+class TransformationClient(Client):
+  """ Exposes the functionality available in the DIRAC/TransformationManagerHandler
```

Comment:
You can review (remove now?) the content of the docstring below this line

Reply:
Done